### DISCUSSION TITLE
Search for all the accepted Machine translations

### DISCUSSION THREAD

**Discussion by shrihari-k** [WeblateOrg/weblate#10063] [Q&A]
Hi,
I am looking for a search query to pull all the translations that was accepted / cloned based on Machine translation so that we can review the same. Is there a way to pull from search with a query?

**Comment by nijel** [comment]
Not easily possible right now, but we're working on a solution for https://github.com/WeblateOrg/weblate/issues/3009 that will make all machine translation have a dedicated user which you can search for. See https://github.com/WeblateOrg/weblate/pull/10096
